pages/tokens/[type].tsx

```diff
@@ -22,11 +22,11 @@ const NATIVE_TOKEN_TEMPLATE_URL =
   'https://github.com/magickbase/godwoken_explorer/issues/new?assignees=Keith-CY&labels=Token+Registration&template=register-a-new-native-erc20-token.yml&title=%5BNative+ERC20+Token%5D+%2A%2AToken+Name%2A%2A'
 
 const parseTokenName = (name: string) => {
-  const parsed = name?.split(/\(via|from/) ?? []
+  const parsed = name?.match(/Wrapped (\w+) \((\w+) from (\w+)\)/) ?? []
```

Comment:
`Wrapped` is not mandatory in `display name` https://github.com/nervosnetwork/rfcs/blob/a092218b8f3ba9b6616ff41bd56a5a75d42efaf7/rfcs/0000-universal-asset-notation/0000-universal-asset-notation.md